reference-data-proxy/src/v1/controllers/premium-schedule.controller.ts

```diff
@@ -77,10 +88,10 @@ export const getPremiumSchedule = async (req: Request, res: Response) => {
     return res.status(400).send();
   }
 
-  if (postPremiumScheduleResponse === 200 || postPremiumScheduleResponse === 201) {
+  if (successStatus.includes(postPremiumScheduleResponse)) {
```

Comment:
would this return a 400/404/etc status? I'm just looking at the old validation where we were checking for 200 or 201 statues only

Reply:
`successStatus` array only has `200` and `201` statues, actual implementation is still the same.